database/migrations/2018_09_28_140458_add_user_type_to_voucher_states.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddUserTypeToVoucherStates extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('voucher_states', function (Blueprint $table) {
+            $table->string('user_type')
+                ->after('user_id')
+                ->default(""); // there needs to be a default, as this is not a null-able field.
```

Comment:
Annoying - but I think it's better like this than to default to any actual type of user